src/main/java/digital/slovensko/autogram/core/Autogram.java

```diff
@@ -181,17 +189,12 @@ public Batch getBatch(String batchId) {
     public void pickSigningKeyAndThen(Consumer<SigningKey> callback) {
         var drivers = driverDetector.getAvailableDrivers();
         ui.pickTokenDriverAndThen(drivers,
-                (driver) -> requestPasswordAndThen(driver, callback));
+                (driver) -> fetchKeysAndThen(driver, callback));
```

Comment:
Toto sa paradne vycistilo, lebo tu netreba saskovat s tym, ze ci ocakava password alebo neocakava (je to pomerne zlozita logika co uz aj tak v pkcs je)